utils/secureContext.js

```
@@ -0,0 +1,250 @@
+          configurable: false
+        });
+      } catch (err) {
+        console.warn('Could not secure __proto__ access:', err.message);
+      }
+    }
+  }
+  
+  if (freezePrototypes && context.Object) {
+    // Freeze key prototypes if available
+    try {
+      if (context.Object && context.Object.prototype) Object.freeze(context.Object.prototype);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)